manual/scripting/code-examples/scene-objects.cs

```diff
@@ -0,0 +1,23 @@
+using FlaxEngine;
+
+public class MyScript : Script
+{
+    public override void OnStart()
+    {
+        // Prints all children names
+        var children = Actor.GetChildren();
```

Comment:
It should be `var children = Actor.Children`

Reply:
Done